client/pages/product/updateproduct.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { axiosInstance } from '../../configs/axios';
+
+const UpdateProduct = ({ product }) => {
+  const [updatedProduct, setUpdatedProduct] = useState({
+    name: product.name,
+    description: product.description,
+    image: product.image,
+    price: product.price,
+    intervals: product.intervals
+  });
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const token = window.localStorage.getItem('token');
```

Comment:
again why? aren't you using interceptors for this? if this was copy pasted from other files to be reused, I'd suggest cleaning it up as well.